test/Makefile

```diff
@@ -103,7 +103,7 @@ export SEP=$(subst /,\,/)
 DRUNTIME_PATH=..\..\druntime
 PHOBOS_PATH=..\..\phobos
 export DFLAGS=-I$(DRUNTIME_PATH)\import -I$(PHOBOS_PATH)
-export LIB=$(PHOBOS_PATH)
+#export LIB=$(PHOBOS_PATH)
```

Comment:
It overwrites the LIB environment variable, some of the tests have the flag "-conf=" set to nothing. So if this line is allowed the tests won't compile as they won't know where the libraries like shell32.lib are.